modules/hostap/src/supp_main.h

```diff
@@ -15,7 +15,7 @@
 	!defined(CONFIG_EAP_PSK) && !defined(CONFIG_EAP_PAX) && \
 	!defined(CONFIG_EAP_SAKE) && !defined(CONFIG_EAP_GPSK) && \
 	!defined(CONFIG_EAP_PWD) && !defined(CONFIG_EAP_EKE) && \
-	!defined(CONFIG_EAP_IKEV2 && !defined(CONFIG_EAP_GTC)
+	!defined(CONFIG_EAP_IKEV2) && !defined(CONFIG_EAP_GTC)
```

Comment:
Also fixed by https://github.com/zephyrproject-rtos/zephyr/pull/79974/commits/de54470bac24e8f6d201105a12ff867fd38d383f

Reply:
@jukkar how do you want deal with this? Merge this and abandon that PR or wait for it to be merged and rebase this?

Reply:
I think you should let the other PR still move forward. @jukkar PR is needed to add the test case going forward.